Makefile

```diff
@@ -445,9 +445,23 @@ release-manifests:
 	cp metadata.yaml $(RELEASE_DIR)/metadata.yaml
 
 .PHONY: release-changelog
-release-changelog: $(RELEASE_NOTES) check-release-tag check-previous-release-tag check-github-token $(RELEASE_DIR) ## Builds the changelog for a release
-	$(RELEASE_NOTES) --debug --org $(GH_ORG_NAME) --repo $(GH_REPO_NAME) --start-sha $(shell git rev-list -n 1 ${PREVIOUS_VERSION}) --end-sha $(shell git rev-list -n 1 ${RELEASE_TAG}) --output $(RELEASE_DIR)/CHANGELOG.md --go-template go-template:$(REPO_ROOT)/hack/changelog.tpl --dependencies=false --branch=$(BRANCH)
+release-changelog: #Generates release notes using Github release notes.
+    $(GH) api repos/$(GH_ORG_NAME)/$(GH_REPO_NAME)/releases/generate-notes -F tag_name=${RELEASE_TAG} --template \
+'{{printf "# Release notes for Cluster API Provider AWS (CAPA) '"$VERSION"'"}}
```

Comment:
Would it be better if we had "template" files for the header and footer instead? I think it would be easier for editing and also less noise for the makefile. What do you think?